### PR TITLE
docs: train api

### PR DESCRIPTION
**Pull request by RobDeFlop**

### CODE REVIEW

File: docs/articles/toc.yml [hunk 1]
```diff
@@ -20,6 +20,11 @@
       href: player/server-player-funcs.md
     - name: Local Player ScriptID
       href: player/player-script-id.md
+- name: Vehicle
```
Comment (C0kkie) [inline docs/articles/toc.yml:23]: Move it below snippets

Comment (RobDeFlop) [reply docs/articles/toc.yml]: Done

File: docs/articles/vehicle/train-api.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Train API
+
+The Train API has been introduced in version 7.0 and provides properties and methods to control trains. This article should give you an overview of the API and how to use it.
```
Comment (C0kkie) [inline docs/articles/vehicle/train-api.md:3]: We should write, that its serverside and also synchronized

File: docs/articles/vehicle/train-api.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+| trainCarriageConfigIndex  | Gets or sets the config index of the train's carriage.                                                                                |
+| trainLinkedToBackwardId   | Gets or sets a vehicle that is linked to the back of the train.                                                                       |
+| trainLinkedToForwardId    | Gets or sets a vehicle that is linked to the front of the train.                                                                      |
+
+## Methods
+
+| Method Name                   | Description                                   |
+| ----------------------------- | --------------------------------------------- |
+| setTrainEngineId              | Applying a vehicle as an engine of a train.   |
+| setTrainLinkedToBackwardId    | Links a vehicle to the back of the train.     |
```
Comment (C0kkie) [inline docs/articles/vehicle/train-api.md:30]: "vehicle" is the wrong word for this case afaik

**Comment by C0kkie** [comment]
Thanks

**Pull request merged**